src/main/java/reposense/model/RepoConfiguration.java

```diff
@@ -554,14 +555,18 @@ public void setUntilDate(LocalDateTime untilDate) {
         this.untilDate = untilDate;
     }
 
-    public String getZoneId() {
+    public ZoneId getZoneId() {
         return zoneId;
     }
 
-    public void setZoneId(String zoneId) {
+    public void setZoneId(ZoneId zoneId) {
         this.zoneId = zoneId;
     }
 
+    public void setZoneId(String zoneId) {
+        this.zoneId = ZoneId.of(zoneId);
+    }
+
```

Comment:
I think this function can be removed after removing the usages in `CommitInfoAnalyzerTest.java`.